.github/workflows/deploy-to-control-plane.yml

```
@@ -34,60 +36,100 @@ jobs:
+      - name: Set PR Number for Comment Event
+        if: github.event_name == 'issue_comment'
+        run: |
+          echo "PR_NUMBER=${{ github.event.issue.number }}" >> $GITHUB_ENV
+          echo "APP_NAME=qa-react-webpack-rails-tutorial-pr-${{ github.event.issue.number }}" >> $GITHUB_ENV
+
+      - name: Check if Review App Exists
+        id: check-app
+        if: github.event_name == 'push' && steps.get-pr.outputs.has_pr == 'true'
+        env:
+          CPLN_TOKEN: ${{ secrets.CPLN_TOKEN }}
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:5:27: Double quote to prevent globbing and word splitting

(shellcheck)